Hunting Queries/AzureActivity/Common_Deployed_Resources.yaml

```diff
@@ -16,7 +16,12 @@ relevantTechniques:
 query: |
 
   AzureActivity
+  | where OperationNameValue has_any (@"deployments/write", @"virtualMachines/write")// or OperationNameVaue has  
```

Comment:
may want to remove this comment as it looks incomplete -` // or OperationNameVaue has`

Reply:
Thanks, missed removing that